easybuild/easyconfigs/d/DBD-mysql/DBD-mysql-4.050-foss-2020b-Perl-5.32.0.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'PerlModule'
+
+name = 'DBD-mysql'
+version = '4.050'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'https://metacpan.org/pod/distribution/DBD-mysql/lib/DBD/mysql.pm'
+description = "Perl binding for MySQL"
+
+toolchain = {'name': 'foss', 'version': '2020b'}
```

Comment:
Does this use MPI and/or BLAS? I would suspect not, and that it can be moved to GCC level like MariaDB.

Reply:
moving to GCC